crates/bitwarden-core/src/client/internal.rs

```diff
@@ -213,7 +213,7 @@ impl InternalClient {
     #[cfg(feature = "internal")]
     pub(crate) fn initialize_user_crypto_pin(
         &self,
-        pin_key: MasterKey,
+        pin_key: PinKey,
```

Comment:
It seems we used the wrong type here. This required me to duplicate the functions for UserKey.